cmd/minishift/cmd/env.go

```diff
@@ -68,7 +68,7 @@ func getConfigSet(api libmachine.API, forceShell string, noProxy bool) (*DockerS
 		return nil, err
 	}
 
-	cmdLine := "minishift docker-env"
+	cmdLine := strings.Join(os.Args, " ")
```

Comment:
👍

Reply:
-1 for this approach. Because it will pick anything from the command line arguments e.g. I got below while testing the PR 
```
$ minishift docker-env hekk
export DOCKER_TLS_VERIFY="1"
export DOCKER_HOST="tcp://192.168.64.247:2376"
export DOCKER_CERT_PATH="/Users/lmohanty/.minishift/certs"
# Run this command to configure your shell:
# eval $(minishift docker-env hekk)
```

Reply:
I think you should append `--unset` or `--nop-proxy` to `minishift docker-env` depending upon what is selected.

Reply:
@LalatenduMohanty yes that is the better approach, updating the patch accordingly